src/main/java/uk/gov/hmcts/reform/userprofileapi/util/IdamStatusResolver.java

```diff
@@ -35,6 +35,8 @@ private IdamStatusResolver() {
 
     public static final String ACTIVE = "ACTIVE";
     public static final String PENDING = "PENDING";
+    public static final String IDAM_5XX_ERROR_RESPONSE = "The identity server cannot process "
```

Comment:
Soory @shivajihmcts , I should have corrected this before
can the message be "The identity server is unable to  process the request due to high load. Please try again later"